### DISCUSSION TITLE
How to correctly add a feature request

### DISCUSSION THREAD

**Discussion by osfanbuff63** [EssentialsX/Essentials#4792] [Q&A] [answered]
How would a feature request be best presented? From what I can tell, you should create a discussion, but would it be better as an issue? I'm fine with whatever is most helpful.

**Comment by pop4959** [comment, marked as answer]
Use discussions for general discussion, questions, and feature suggestions, and issues for reporting bugs.

Discussions are pretty helpful for feature requests generally, because it's fairly easy to see which features are popular via the voting system.
